### PR TITLE
suhy/cdx 44 blog initial setup

### PR DESCRIPTION
**Pull request by suhy-jang**
- Create a blog page
- Connect WordPress data (from temporary website)
- Set layouts
- Remove unnecessary codes
- Improve reusability

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -10,6 +10,14 @@ module.exports = {
     siteUrl: `https://codivox.com`,
   },
   plugins: [
+    {
+      resolve: 'gatsby-source-graphql',
+      options: {
+        typeName: 'WPGraphQL',
+        fieldName: 'wpgraphql',
+        url: 'http://codivox-blog-wp-storage.herokuapp.com/graphql',
```
Comment (inhaq) [inline gatsby-config.js:18]: please use `https`

Comment (suhy-jang) [reply gatsby-config.js:18]: Although I use https, WordPress has saving images at http, mixed-content issue will be remain.
https://github.com/Codivox/codivox.com/pull/54/checks?check_run_id=1095037405

**Review by inhaq — APPROVED** [review]
Awesome 👍🏻

**Comment by suhy-jang** [comment]
Thank you for review, @inhaq

**Pull request merged**